workers/ohsome_quality_analyst/api.py

```diff
@@ -211,6 +217,42 @@ async def _fetch_report(
     return response
 
 
-@app.get("/regions")
+@app.get("/list_regions")
```

Comment:
Imho that's a breaking change. Please label/sort that accordingly in the CHANGELOG.md.